### PR TITLE
Comments

### PR DESCRIPTION
**Pull request by mikeblas**

### CODE REVIEW

File: src/Docker.DotNet/Endpoints/IContainerOperations.cs [hunk 1]
```diff
@@ -73,7 +73,7 @@ public interface IContainerOperations
         /// <param name="cancellationToken">When triggered, the operation will stop at the next available time, if possible.</param>
         /// <returns>A <see cref="Task{TResult}"/> that resolves to a <see cref="Stream"/>, which provides the log information.</returns>
         /// <remarks>This method works only for containers with the <c>json-file</c> or <c>journald</c> logging driver.
-        /// <br/>The corresponding commands in the Docker CLI are <c>docker logs</c> and <c>docker container logs</c>.</remarks>
+        /// <br/>The corresponding command in the Docker CLI is <c>docker logs</c>.</remarks>
```
Comment (HofmeisterAn) [inline src/Docker.DotNet/Endpoints/IContainerOperations.cs:76]:
```suggestion
        /// <br/>The corresponding commands in the Docker CLI are <c>docker logs</c> and <c>docker container logs</c>.</remarks>
```

Both are valid Docker CLI commands.

File: src/Docker.DotNet/Endpoints/IContainerOperations.cs [hunk 2]
```diff
@@ -91,8 +91,7 @@ public interface IContainerOperations
         /// <returns>A <see cref="Task"/> that will complete once all log lines have been read, or once the container has exited if Follow is set to <see langword="true"/>.</returns>
         /// <remarks>This method is only suited for containers created with a TTY. For containers created without a TTY, use
         /// <see cref="GetContainerLogsAsync(string, bool, ContainerLogsParameters, CancellationToken)"/> instead.
-        /// <br/>
-        /// The corresponding commands in the Docker CLI are <c>docker inspect</c> and <c>docker container inspect</c>.</remarks>
+        /// <br/>The corresponding command in the Docker CLI is <c>docker logs</c>.</remarks>
```
Comment (HofmeisterAn) [inline src/Docker.DotNet/Endpoints/IContainerOperations.cs:94]:
```suggestion
        /// <br/>The corresponding commands in the Docker CLI are <c>docker logs</c> and <c>docker container logs</c>.</remarks>
```

File: src/Docker.DotNet/Endpoints/IContainerOperations.cs [hunk 3]
```diff
@@ -108,7 +107,7 @@ public interface IContainerOperations
         /// <param name="cancellationToken">When triggered, the operation will stop at the next available time, if possible.</param>
         /// <returns>A <see cref="Task{TResult}"/> that resolves to a <see cref="MultiplexedStream"/>, which provides the log information.
         /// If the container wasn't created with a TTY, this stream is multiplexed.</returns>
-        /// <remarks>The corresponding commands in the Docker CLI are <c>docker inspect</c> and <c>docker container inspect</c>.</remarks>
+        /// <remarks>The corresponding command in the Docker CLI is <c>docker logs</c>.</remarks>
```
Comment (HofmeisterAn) [inline src/Docker.DotNet/Endpoints/IContainerOperations.cs:110]:
```suggestion
        /// <remarks>The corresponding commands in the Docker CLI are <c>docker logs</c> and <c>docker container logs</c>.</remarks>
```

**Review by HofmeisterAn — CHANGES_REQUESTED** [review]

File: src/Docker.DotNet/Endpoints/IContainerOperations.cs (hunk 1 above)
Comment (mikeblas) [reply src/Docker.DotNet/Endpoints/IContainerOperations.cs]: fixed

File: src/Docker.DotNet/Endpoints/IContainerOperations.cs (hunk 2 above)
Comment (mikeblas) [reply src/Docker.DotNet/Endpoints/IContainerOperations.cs]: fixed

File: src/Docker.DotNet/Endpoints/IContainerOperations.cs (hunk 3 above)
Comment (mikeblas) [reply src/Docker.DotNet/Endpoints/IContainerOperations.cs]: fixed

File: src/Docker.DotNet/Endpoints/IContainerOperations.cs [hunk 4]
```diff
@@ -73,7 +73,7 @@ public interface IContainerOperations
         /// <param name="cancellationToken">When triggered, the operation will stop at the next available time, if possible.</param>
         /// <returns>A <see cref="Task{TResult}"/> that resolves to a <see cref="Stream"/>, which provides the log information.</returns>
         /// <remarks>This method works only for containers with the <c>json-file</c> or <c>journald</c> logging driver.
-        /// <br/>The corresponding commands in the Docker CLI are <c>docker logs</c> and <c>docker container logs</c>.</remarks>
+        /// <br/>The corresponding command in the Docker CLI are </c> <c>docker logs</c> and <c>docker container logs</c>.</remarks>
```
Comment (HofmeisterAn) [inline src/Docker.DotNet/Endpoints/IContainerOperations.cs:76]:
```suggestion
        /// <br/>The corresponding command in the Docker CLI are <c>docker logs</c> and <c>docker container logs</c>.</remarks>
```

File: src/Docker.DotNet/Endpoints/IContainerOperations.cs [hunk 5]
```diff
@@ -91,8 +91,7 @@ public interface IContainerOperations
         /// <returns>A <see cref="Task"/> that will complete once all log lines have been read, or once the container has exited if Follow is set to <see langword="true"/>.</returns>
         /// <remarks>This method is only suited for containers created with a TTY. For containers created without a TTY, use
         /// <see cref="GetContainerLogsAsync(string, bool, ContainerLogsParameters, CancellationToken)"/> instead.
-        /// <br/>
-        /// The corresponding commands in the Docker CLI are <c>docker inspect</c> and <c>docker container inspect</c>.</remarks>
+        /// <br/>The corresponding command in the Docker CLI are </c> <c>docker logs</c> and <c>docker container logs</c>.</remarks>
```
Comment (HofmeisterAn) [inline src/Docker.DotNet/Endpoints/IContainerOperations.cs:94]:
```suggestion
        /// <br/>The corresponding command in the Docker CLI are <c>docker logs</c> and <c>docker container logs</c>.</remarks>
```

File: src/Docker.DotNet/Endpoints/IContainerOperations.cs [hunk 6]
```diff
@@ -108,7 +107,7 @@ public interface IContainerOperations
         /// <param name="cancellationToken">When triggered, the operation will stop at the next available time, if possible.</param>
         /// <returns>A <see cref="Task{TResult}"/> that resolves to a <see cref="MultiplexedStream"/>, which provides the log information.
         /// If the container wasn't created with a TTY, this stream is multiplexed.</returns>
-        /// <remarks>The corresponding commands in the Docker CLI are <c>docker inspect</c> and <c>docker container inspect</c>.</remarks>
+        /// <br/>The corresponding command in the Docker CLI are </c> <c>docker logs</c> and <c>docker container logs</c>.</remarks>
```
Comment (HofmeisterAn) [inline src/Docker.DotNet/Endpoints/IContainerOperations.cs:110]:
```suggestion
        /// <remarks>The corresponding command in the Docker CLI are <c>docker logs</c> and <c>docker container logs</c>.</remarks>
```

**Review by HofmeisterAn — CHANGES_REQUESTED** [review]

**Pull request merged**